config.toml

```diff
@@ -44,5 +44,5 @@ back = "Indietro"
 contact = "Contattaci"
 donate = "Dona"
 home = "Pagina Iniziale"
-news = "Notizie"
+news = "Notizie (en)"
```

Comment:
These (en) are not needed. Users can enable google translate if necessary.

Reply:
`/news` redirects to English version from web-site without any way to get back. It looks a but confusing. Anyway, I removed "(en)".